app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,21 @@
 <div class="menu">
 <a href="/" class="text-light"><svg xmlns="http://www.w3.org/2000/svg" width="26" height="36" fill="currentColor" class="bi bi-arrow-left" viewBox="0 0 16 16">
-  <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
-</svg></a>
+     <path fill-rule="evenodd" d="M15 8a.5.5 0 0 0-.5-.5H2.707l3.147-3.146a.5.5 0 1 0-.708-.708l-4 4a.5.5 0 0 0 0 .708l4 4a.5.5 0 0 0 .708-.708L2.707 8.5H14.5A.5.5 0 0 0 15 8z"/>
+   </svg>
+</a>
 <h2 class="text-light text-center">Log In</h2>
 <%= link_to "Sign Up", new_user_registration_path, class: "text-light text-center"%>
 </div>
-<div class="col-md-4 offset-md-4 col-sm-8 offset-sm-2 mt-5">
- 
+<div class="col-md-4 offset-md-4 col-sm-8 offset-sm-2 mt-5"> 
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-   
-     
-    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email",  class:"form-control mt-5 mb-2 opacity-75" %>  
-     
-    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class:"form-control mt-4 mb-2 opacity-75" %> 
-
-  <% if devise_mapping.rememberable? %> 
-      <%= f.check_box :remember_me %>
-      <%= f.label :remember_me %> <br />
-  <% end %> 
-   
-  <%= f.submit "Log in", class:"btn btn-outline-primary fw-bold mt-5 center" %>
-  
+    <div class="bg-light mt-5 p-3">
+    <%= f.email_field :email, autocomplete: "email", placeholder: "Email",  class:"form-control opacity-75" %>
+    <hr/> 
+    
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password", class:"form-control opacity-75 back" %> 
+    </div>
+    <%= f.submit "Log in", class:"btn btn-primary log-btn1 fw-bold mt-5 center" %>
+    
 <% end %>
 <h4 class=" text-center log_link"><%= render "devise/shared/links" %></h4>
 </div>
```

Comment:
- [ ] @AhmadiGu great work implementing the changes requested however the design still doesn't match the behance design given, the forgot passowrd button is bigger than expected also remove the background color in the input fields

Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/232227361-75356395-eed5-4b62-ac7c-28642fdcbba4.png) |  ![image](https://user-images.githubusercontent.com/61408860/232227398-bad552ae-a289-43d5-b086-98c3af70b9f4.png)